shimmingtoolbox/masking/mask_mrs.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+"""
+Creating MRS mask API
+"""
+import numpy as np
+import numpy.linalg as npl
+import nibabel as nib
+import logging
+from shimmingtoolbox.utils import splitext
+from shimmingtoolbox.utils import run_subprocess
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+def mask_mrs(fname_input, raw_data, center, size):
+    """
+    Create a mask to shim single voxel MRS
+
+    Args:
+        fname_input (str): Input path of the fieldmap to be shimmed (supported extention .nii and .nii.gz) [required]
+        raw_data (str): Input path of the of the twix raw-data (supported extention .dat) [optional]
+        center (float): voxel's center position in mm of the x, y and z of the scanner's coordinate [optional]
+        size (float): voxel size in mm of the x, y and z of the scanner's coordinate [optional]
+
+    Returns:
+        numpy.ndarray: Cubic mask with same dimensions as MRS voxel.
+    """
+
+    if fname_input is None:
+        raise TypeError(f"The fname_input is cannot be None. See: st_mask mrs-mask -h")
```

Comment:
This is the API, you are referencing something that is outside its scope (the CLI) by saying `st_mask mrs-mask -h`.  You should do this in the CLI file if you want tot write this error message. The API should contain this type of error checking, but with a different comment that makes sense to its scope.